ruleset.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<ruleset name="Sdk">
+    <description>
+        Spryker Coding Standard for Project.
+
+        Extends main Spryker Coding Standard.
+        All sniffs in ./Sniffs will be auto loaded
+    </description>
+
+    <rule ref="vendor/spryker/code-sniffer/Spryker/ruleset.xml"/>
+
+    <file>./src</file>
```

Comment:
we usually define them as src/ instead (trailing slash on folders)
See https://github.com/spryker/decimal-object/blob/master/phpcs.xml#L13-L14
and our docs